app/views/courses/age_range/_errors.html.erb

```diff
@@ -0,0 +1,23 @@
+<% error_summary_title ||= 'You’ll need to correct some information.' %>
+<% if @errors && @errors.any? %>
+  <div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="govuk-error-summary" data-ga-event-form="error">
+    <h2 class="govuk-error-summary__title" id="error-summary-title">
+      <%= error_summary_title %>
+    </h2>
+    <div class="govuk-error-summary__body">
+      <ul class="govuk-list govuk-error-summary__list">
+        <% if @errors.key?(:age_range_in_years) %>
```

Comment:
The error handling for this form is now a bit more complicated and required re-creating the error partial. If the user fails to fill in both the 'From' and 'To' fields then we only want the first of the three error messages shown ("Enter an age in both From and To")

Reply:
just wonder if the `shared` error partial can we inject this little bit of html...

a bit like those `form_fields`

Reply:
Mmm ... Not sure if that is the right thing to do as this bit of logic relates specifically to age range?